src/main.js

```diff
@@ -14,14 +14,16 @@ import Login from './pages/Login.vue'
 
 // Import components
 import Masthead from './components/Masthead.vue'
+import Logo from './components/Logo.vue'
 
 
 // Define routes
 const routes = [
+{ path: '/', component: Account },
 { path: '/account', component: Account },
```

Comment:
Yeah, I just chose a page which has the navbar on it - but it ought to be the Add a voucher page though, since that's what will display after logging in. I'll change it to that.